hello-world/tutorial/index.md

```diff
@@ -45,6 +45,16 @@ To install `starport` into `/usr/local/bin`, run the following command:
 curl https://get.starport.network/starport@v0.16.2! | bash
 ```
 
+In case the above command does not work, use the following:
+
+```
+curl https://get.starport.network/starport > starport.sh
```

Comment:
```suggestion
curl https://get.starport.network/starport@v0.16.2! > starport.sh
```

Reply:
if we don't specify a version, then the latest version is installed, and the tutorials are tested and supported only for the specified versions